src/FoodStorage/foodStorage.py

```diff
@@ -1,16 +1,30 @@
+from src.Constants.constants import \
+    ADD_FOODS_SUCCESS_MESSAGE, \
```

Comment:
Why did you have to import every constant individually?

Reply:
So I guess for here, it looks like it doesn't make sense to import everything individually because we're using everything. But if we were adding more things to the constants file and only using a part of it here, we only want to import the things we use. I guess I can change it to import all for now though and we can change it to individual variables if we end up adding more to the constants file.